posthog/templates/email/member_join.html

```diff
@@ -0,0 +1,20 @@
+{% extends "email/base_personal.html" %} {% load posthog_assets %} {% load posthog_filters %}
+{% block heading %}{{ invitee.first_name }} accepted their PostHog invitation{% endblock %}
+{% block section %}
+    <p>
+        {{ invitee.first_name }} (<a href="mailto:{{ invitee.email }}">{{ invitee.email }}</a>) just joined you at {{ organization.name }}.
+    </p>
+    <p>
+        PostHog is a product analytics platform made to help you understand your users.<br/>
```

Comment:
Only comment: I would remove this line. If you're getting this message, you already know what PostHog is.